examples/seismic/tti/tti_example.py

```diff
@@ -81,8 +74,15 @@ def test_tti_stability(shape, kernel):
                         help="Choice of finite-difference kernel")
     args = parser.parse_args()
 
-    # Switch to TTI kernel if input is acoustic kernel
-    preset = 'layers-tti-noazimuth' if args.azi else 'layers-tti'
+    preset = 'layers-tti'
```

Comment:
I would maybe rather move this inside the nested `else`

```
if ...
    ...
else:
    if args.azi:
        ....
    else:
        preset = 'layers-tti'
```

Reply:
Ok